### DISCUSSION TITLE
erros from JULES5.0

### DISCUSSION THREAD

**Discussion by jachungithub** [NASA-LIS/LISF#1149] [Q&A]
As suggested, I created a JULES ancillary file (ancil4jules.nc), but from some other JULES off-line configurations (not LIS) and generated lis_input.d01.nc by LDT (JULES5.0, LISF-public-7.3.3). However, the LIS run (JULES5.0) couldn't be finished with an error (lislog.0000: "[ERR] No dominant surface model type found!"). Based on LIS/LDT source codes, I think the error message indicates that the lis_input.d01.nc file might not be correctly generated. I also found that "[INFO] cannot access dataset igbp in ./ancil4jules.nc" in ldtlog.0000, but it says "[INFO] Done reading ./ancil4jules.nc". This pattern can be also seen at the reading of the other parameters in ancil4jules.nc.

I attached the mentioned files above (erros_JULES50.zip). Any suggestions would be greatly appreciated.
Furthermore, anyone can share the ancillary nc file (cap2ldt_ps41.nc) with me, that would be appreciated. I would like to check my ancillary file by myself against cap2ldt_ps41.nc.

[erros_JULES50.zip](https://github.com/NASA-LIS/LISF/files/9127236/erros_JULES50.zip)

**Comment by sanjaync** [comment]
Hi @jachungithub 
did you solve this error, i am getting same error in LIS run for VIC model 4.1.2 . 
the error is as follows: 
 [INFO] Reading soiltexture map from ./lis412param_org.nc
 [ERR] No dominant surface model type found!
 c,r,fgrd:            1           3   0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000       0.00000000    
 [ERR] endrun is being called


Can you suggest any solution to solve this error please.
